mitodl/open-discussions
# Remove pyopenssl

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #3829, replaces #3825 

#### What's this PR do?
Removes pyopenssl and updates cryptography

#### How should this be manually tested?
You should be able to register new users and sign in.  Test with a local edx instance and make sure that user integration still works.


#### Any background context you want to provide?
https://pypi.org/project/pyOpenSSL/
